exts/morse.py

```diff
@@ -25,13 +25,14 @@ def __init__(self, bot: commands.Bot):
     @commands.command(name="morse", aliases=["cw"], category=cmn.Cats.CODES)
     async def _morse(self, ctx: commands.Context, *, msg: str):
         """Converts ASCII to international morse code."""
-        result = ""
+        result = "`"
         for char in msg.upper():
             try:
                 result += self.morse[char]
             except KeyError:
                 result += "<?>"
             result += " "
+        result += "`"
         embed = cmn.embed_factory(ctx)
         embed.title = f"Morse Code for {msg}"
         embed.description = "**" + result + "**"
```

Comment:
instead of adding it earlier, add the codequotes here

Reply:
On line 38? How does it interact with `**`?

Reply:
Oh I think I see, the morse output specifically is bolded for visibility?

Reply:
![image](https://github.com/user-attachments/assets/2d0dfb51-aeab-424b-a0f5-3724b6a9d731)
 correct